.travis.yml

```diff
@@ -77,3 +77,4 @@ script:
 branches:
     only:
         - master
+        - /ci\/.*/
```

Comment:
I've added this so I can push branches to my fork and get Travis to double-check my WIPs. I've not been able to find a less invasive way to do that short of carrying this change around, so if that's not wanted it can be dropped from the PR.